portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,78 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Ibiyemi's Portfolio</title>
+  <link
```

Comment:
Best practice for a multiline html tag is to have one attribute on the first line and horizontally align the others to it:
```
<foo bar="baz"
     zoop="doop"
     hello="world">
```

Reply:
Please fix all instances.

Reply:
Done